Queue/Queue.cpp

```diff
@@ -0,0 +1,86 @@
+#include"Queue.h"
+using namespace std;
+
+void CreateQueue(Queue& q, int size)
+{
+	q.data = new int[size];
```

Comment:
I don't see you free the data after ending your program. Can you run Valgrind to check if we are leaking the memory or not?